src/main/resources/icons/files/lint-staged.svg

```diff
@@ -0,0 +1,29 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32">
```

Comment:
I don't like this icon. This is a combination of emoji, not a genuine logo. Let's wait until a designer creates a logo for that or find something more suited (something like a broom can be good)

Reply:
Me too but there is no icon for this package nowadays except of this. Also this one is used by `vscode-icons` library. Fine. I will try to find something interesting.

Reply:
It's so hard to find something reliable. Idk may be something simple like [this](https://www.flaticon.com/free-icon/broom_1159062#term=broom%20flat&page=1&position=10) or will wait?

Reply:
This could do, but if you use it try keeping only one color

Reply:
Check please